BrainPortal/lib/scir_unix.rb

```diff
@@ -35,11 +35,11 @@ def update_job_info_cache #:nodoc:
       @job_info_cache = {}
       ps_command = case CBRAIN::System_Uname
         when /Linux/i
-          "ps ax -o pid,uid,state"
+          "ps x -o pid,uid,state"
         when /Solaris/i
-          "ps ax -o pid,uid,state"  # not tested
+          "ps x -o pid,uid,state"  # not tested
         else
-          "ps ax -o pid,uid,state"  # not tested
+          "ps x -o pid,uid,state"  # not tested
```

Comment:
Pierre, the ps command line for Linux, Solaris and other systems is identical. Shall we remove the case statement and keep only 1 command line?